src/renderers/WebGLRenderer.js

```diff
@@ -2719,8 +2719,9 @@ function WebGLRenderer( parameters ) {
 
 	this.copyFramebufferToTexture = function ( position, texture, level ) {
 
-		var width = texture.image.width;
-		var height = texture.image.height;
+		var levelScale = Math.pow( 2, - level );
+		var width = texture.image.width * levelScale;
+		var height = texture.image.height * levelScale;
```

Comment:
Maybe `var width = Math.floor( texture.image.width * levelScale );` in order to force integer values?